include/libm2k/context.hpp

```diff
@@ -75,7 +75,7 @@ class LIBM2K_API Context {
 	std::string getContextAttributeValue(std::string attr);
```

Comment:
this commit is missing a `Signed-off-by` tag;
do we need/want to use this for this project? different projects have different rules and legal-stuff
if yes, then it may be an idea to add it;

Reply:
We should follow the sign-off process on all our repositories.
With this tag you certify the Developer's Certificate of Origin.

Please take a look here: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/process/submitting-patches.rst?id=HEAD#n418

Reply:
Done.